### PR TITLE
Convert banner to field warning

### PR DESCRIPTION
**Pull request by edg2s**

### CODE REVIEW

**Comment by edg2s** [comment]
Before:
image
After:
image

**Comment by MatmaRex** [comment]
I know it's a small change, but I really hate it.

**Comment by edg2s** [comment]
> I know it's a small change, but I really hate it.

Tell me what you really think though.

**Comment by edg2s** [comment]
What about an info icon? I think it should probably match the width of the form either way?

**Comment by MatmaRex** [comment]
Really please no

If it should match the width of the form, then I think the form should be allowed to be wider. I'm actually not sure why its width was limited, when the width of the whole page is also already limited? I feel like it would be fine if it was expanded from `50em` to `1100px`, like the table and everything else.

**Comment by edg2s** [comment]
> If it should match the width of the form, then I think the form should be allowed to be wider. I'm actually not sure why its width was limited, 

Forms becomes harder to use when you make them too wide. Either the labels are too far from the inputs, or the inputs are too wide and the select handle is too far from the labels.

File: index.css [hunk 1]
```diff
@@ -86,6 +86,22 @@ form {
 	max-width: 50em;
 }
 
+/* T263737 */
```
Comment (MatmaRex) [inline index.css:89]: No longer needed?

**Review by MatmaRex — CHANGES_REQUESTED** [review]

**Comment by MatmaRex** [comment]
I'm still not convinced that this is an improvement.

**Comment by MatmaRex** [comment]
Ah, whatever.

**Pull request merged**